dapps/profile.vue/src/i18n/en.json

```
@@ -148,6 +149,21 @@
         }
       }
     },
+    "did": {
+      "choose-contact": "Choose a contact",
+      "created-at": "created at",
+      "delegates-desc": "Below you can specify the delegates, who may act on behalf of this identity. This may empower them to edit information, manage sharings or perform transactions.",
+      "delegates-title": "Delegates",
+      "description": "With Decentralized Identity (DID) you can identify yourself uniquely, globally and independently of the network or platform. You may also define, who is allowed to make changes to this identity (controller) and who receives authorization to act on behalf of it (delegates). Exporting this document enables you to share all the information with others offline.",
```

```suggestion
      "description": "With Decentralized Identities (DID) you can identify yourself uniquely, globally and independently of the network or platform. You may also define, who is allowed to make changes to this identity (controller) and who receives authorization to act on behalf of it (delegates). Exporting this document enables you to share all the information with others offline.",
```

used the plural also in the german translation